Gemfile

```diff
@@ -130,3 +130,4 @@ gem 'omniauth-github', '~> 1.1', '>= 1.1.2'
 gem 'activerecord-tableless'
 gem 'figaro'
 gem 'sanitize'
+gem 'openssl', '~> 2.0.0.beta.1'
```

Comment:
Could we move this into the `development` gems only, since we won't be using this in production? Thanks!!!

Reply:
yes for sure.